pkg/lns/chirpstack.go

```diff
@@ -121,7 +121,7 @@ func (c *ChirpstackClient) Listen(ch chan<- types.DeviceEvent) {
 								fmt.Printf("[lns][%s] device created %s\n", msg.ID, pl.Metadata["dev_eui"])
 								devices, err := c.GetDevice(pl.Metadata["dev_eui"])
 								if err != nil {
-									log.Printf("Error while creating device :: %s\n", err)
+									fmt.Printf("[lns][error][%s] error creating device lns; skipping helium update for device create: %s :: %s\n", msg.ID, pl.Metadata["dev_eui"], err)
```

Comment:
`error creating device lns; ` <= should be `error fetching device from lns` (we only listen for events here; we received the device create event from Chirpstack, tried to fetch the device; but that failed)

Reply:
Copy/Paste is hard :-P